src/common/util/useCurrentPerson.ts

```diff
@@ -14,7 +14,7 @@ type CurrentPerson = {
 export function useCurrentPerson() {
   const { keycloak } = useKeycloak<KeycloakInstance>()
   return useQuery<CurrentPerson>(
-    endpoints.account.me.url,
+    endpoints.account?.me?.url,
```

Comment:
My whole profile page was blowing up without this change here? Any idea what it might be and is it okay to leave it like that?

Reply:
Seems to be working fine now though 🤨, so I removed the conditionals. If I see this error again I will mention it.